libraries/stdlib/samples/test/samples/misc/tuples.kt

```diff
@@ -20,4 +20,23 @@ class Tuples {
         assertPrints(c, "[null]")
     }
 
+    @Sample
+    fun pairToList() {
+        val list = Pair(1, "a").toList()
+        assertPrints(list, "[1, a]")
+        assertTrue(list[0] is Int)
+        assertTrue(list[1] is String)
+        assertFalse(list[1] is Int)
```

Comment:
I'm not sure this assertion adds something, after you've asserted that this element is `String`